js/graph_spectrum_calc.js

```diff
@@ -40,10 +40,6 @@ GraphSpectrumCalc.initialize = function(flightLog, sysConfig) {
     let minTime = this._flightLog.getMinTime(),
         maxTime = this._flightLog.getMaxTime();
     let timeRange = maxTime - minTime;
-    if (timeRange > MAX_ANALYSER_LENGTH) {
-        maxTime = minTime + MAX_ANALYSER_LENGTH;
-        timeRange = MAX_ANALYSER_LENGTH;
```

Comment:
Should `timeRange` not be constrained as still used on line 45 ?

Reply:
I compute full records count in the logs during of log loading. Therefore i need to use full log time. It will be wrong log rate if else.